chainspec.go

```diff
@@ -0,0 +1,141 @@
+package ibctest
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"strings"
+	"sync"
+	"sync/atomic"
+
+	"github.com/strangelove-ventures/ibctest/ibc"
+)
+
+// ChainSpec is a wrapper around an ibc.ChainConfig
+// that allows callers to easily reference one of the built-in chain configs
```

Comment:
nit:

Curious if `predefined` vs. `built-in` may be clearer?

`predefined chain config` vs. `built-in`. The word "built-in" wasn't as clear to me. But I'm perhaps overthinking this.